packages/server-admin-ui/src/views/DataBrowser.js

```diff
@@ -318,7 +339,37 @@ class DataBrowser extends Component {
                   null,
                   typeof data.value === 'object' && Object.keys(data.value || {}).length > 1 ? 2 : 0)
                 const meta = this.state.meta[this.state.context][data.path]
-                const units = meta && meta.units ? meta.units : ''
+                const valueIsNumber = typeof data.value === 'number' ? true : false //do not show or edit units for objects
+                const units = meta && meta.units ? meta.units : !valueIsNumber ? '' : <Input
+                        type='select'
+                        name='unit'
+                        value={this.state.unit }
+                        onChange={this.handleUnitChange}
+                        >
+                        <option value=''>no unit</option>
+                        <option value='A'>A (Amperes)</option>
```

Comment:
Just an idea that these could come from the server and could be available for plugins that way also.

Reply:
I agree. I actually found them in an old version of the spec pdf...

Reply:
Aha, https://github.com/SignalK/specification/blob/master/schemas/definitions.json#L87 seems to have them